modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/restapi/ContentSearchTestCase.java

```diff
@@ -314,19 +308,39 @@ public void testContentSearchWithAccessControl() throws Exception {
             "wso2.am" }, description = "Test content Search with store visibility", dependsOnMethods = "testContentSearchWithAccessControl")
     public void testContentSearchWithStoreVisibility() throws Exception {
 
+        // Set store visibility to role1
+        HttpResponse httpResponse = restAPIPublisher.getAPI(apiId);
+        Gson g = new Gson();
```

Comment:
```suggestion
        APIDTO apiDto = new Gson().fromJson(httpResponse.getData(), APIDTO.class);
```

Reply:
Fixed in [01a7917](https://github.com/wso2/product-apim/pull/13507/commits/01a7917f0e1323d2c8c94b0ffa1f7c71d673a9d5)